code/game/machinery/computer/warrant.dm

```diff
@@ -0,0 +1,149 @@
+/obj/machinery/computer/warrant//TODO:SANITY
+	name = "security warrant console"
+	desc = "Used to view crewmember security records"
+	icon_screen = "security"
+	icon_keyboard = "security_key"
+	circuit = /obj/item/circuitboard/computer/warrant
+	light_color = LIGHT_COLOR_RED
+	var/authenticated = null
+	var/screen = null
+	var/datum/data/record/current = null
+
+/obj/machinery/computer/warrant/ui_interact(mob/user)
```

Comment:
This should really be done via tgui

Reply:
I'm not making him convert it when the original PR didn't have it.